src/main/kotlin/com/viartemev/requestmapper/contributor/JavaRequestMappingContributor.kt

```diff
@@ -0,0 +1,5 @@
+package com.viartemev.requestmapper.contributor
+
+import com.viartemev.requestmapper.JavaAnnotationSearcher
+
+class JavaRequestMappingContributor : RequestMappingByNameContributor(listOf(JavaAnnotationSearcher::search))
```

Comment:
Should we pass a single function to `RequestMappingByNameContributor`?  
Have you thought about making `RequestMappingByNameContributor` abstract and adding one method for searching annotations? 
In this case, we can delete `Kotlin/JavaAnnotationSearcher` and put this logic in contributor.

Reply:
Fixed. You are right. Thank you!